g7_declaration/SQ1-1j-i.yml

```diff
@@ -7,6 +7,8 @@ options:
     label: licensed
   -
     label: a member of a relevant organisation
+  -
+    label: none of the above    
```

Comment:
This will require a change in the validation code to work - otherwise even if you click none of the above you'll get the message "You must answer this question because you answered 'licenced' or 'a member of a relevant organisation' to question 29" for Q30.